docs: review for OCM plugin (#291)
janus-idp/backstage-plugins: plugins/ocm-backend/README.md

# Open Cluster Management plugin for Backstage

The Open Cluster Management (OCM) plugin integrates your Backstage instance with Open Cluster Management.

> For OCM plugin documentation, see [Open Cluster Management plugin documentation](https://github.com/janus-idp/backstage-plugins/tree/main/plugins/ocm) on GitHub.

Reviewer: ```suggestion
For more information about the OCM plugin, see the [Open Cluster Management plugin documentation](https://github.com/janus-idp/backstage-plugins/tree/main/plugins/ocm) on GitHub.
```

Author: Done

## Development setup

> To run a development setup use the following command:

Reviewer: ```suggestion
You can run a development setup by using the command:
```

Author: Done

```sh
yarn start
```

> The previous command mocks the Kubernetes API to provide two clusters including cluster `foo` which is the hub and cluster `cluster1`. An error response is also mocked for the cluster named `non_existent_cluster`.

Reviewer: Better explanation about these clusters would be helpful - what is `cluster1` for? Also here we make reference to two clusters, but then also mention a cluster named `non_existent_cluster`, so it feels like 3 clusters? It's not clear what these are for, what the relationship is, or whether a cluster named `non_existent_cluster` is intentionally or a possible error that occurs when creating one of the other clusters.

Author: ocm-backend is just a part of the OCM plugin. More information about clusters is available in the ocm README. We have added the reference of that README in this one.

Author: Also, I've changed this as follows, which is more clear:

_When you run the previous command, the Kubernetes API is mocked to provide two clusters: ' foo` (works as the hub) and `cluster1`. Also, an error response is mocked for non-existent clusters._

Reviewer: Much better, thank you!
